bin/circleci/invalidate-cloudfront-cache.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+DISTRIBUTION_ID=$1
+
+aws configure set preview.cloudfront true
+
+INVALIDATION_ID=$(aws cloudfront create-invalidation \
+  --distribution-id $DISTRIBUTION_ID \
+  --paths '/*' | jq -r '.Invalidation.Id');
```

Comment:
Just for my own understanding, I looked up what this jq command does:

jq is 'a lightweight and flexible command-line JSON processor.'

Given input {"Invalidation": {"Id": "foo"}} jq '.Invalidation.Id returns "foo". Adding the -r makes it return foo without the quotes.